x-pack/plugins/security_solution/server/endpoint/services/actions/action_details_by_id.ts

```diff
@@ -52,11 +52,7 @@ export const getActionDetailsById = async (
             body: {
               query: {
                 bool: {
-                  filter: [
-                    { term: { action_id: actionId } },
-                    { term: { input_type: 'endpoint' } },
-                    { term: { type: 'INPUT_ACTION' } },
```

Comment:
Assuming we are still writing these other third-party actions with that value, sure.

That field is to direct fleet server/agent to know the payload is for a child integration and not for _itself_. I'm not sure how these third-party actions/connectors work. If it doesn't use fleet server or agent at all, then I guess it wouldn't make sense to populate this field, and removing it from the query makes sense.

Reply:
The documents that are written do still set `type: 'INPUT_ACTION'`, but I did not see a reason for including it. If I had to guess, it is left over from code we had associated with Fleet indexes, where (I guess 🤔 ) there could be different `type`'s. Since this is our own index and we have control on what is placed there, it did not make sense to me to keep it.

Reply:
Ah yes! This is only indexing/searching on our own index. Then we don't need to filter other than `action_id`.